app/tests/unit/pages/project/project-revision-change-logs.test.tsx

```diff
@@ -103,5 +103,4 @@ describe("The project amendments and revisions page", () => {
     expect(operation.fragment.node.name).toBe("projectRevisionChangeLogsQuery");
     expect(operation.fragment.variables.revisionType).toBe("Minor");
   });
-  // TODO: Add tests when `New Revision` button is implemented
```

Comment:
This comment is still relevant - we need to check that clicking on the "new revision" button calls the right mutation